avm/res/insights/private-link-scope/main.json

```diff
@@ -0,0 +1,1315 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "languageVersion": "2.0",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.23.1.45101",
```

Comment:
Minor, but would be great to generate with latest Bicep version

Reply:
done